pynetbox_data_uploader/lib/utils/query_dataclass.py

```diff
@@ -0,0 +1,39 @@
+from typing import List
+from dataclasses import replace
+from lib.utils.device_dataclass import Device
+from lib.netbox_api.netbox_get_id import NetboxGetId
+
+
+# pylint: disable = R0903
+class QueryDataclass:

```

Comment:
This isn't a dataclass so name it something else

Reply:
👍